apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -161,6 +171,68 @@ export class TeacherFeedback extends Component {
     );
   }
 
+  renderFeedbackTeacherHeader() {
+    // Pilots which the user is enrolled in (such as keep working experiment) are stored on
+    // window.appOptions.experiments, which is queried by experiments.js
+    const keepWorkingEnabled = experiments.isEnabled(keepWorkingExperiment);
+    const latestFeedback = this.state.latestFeedback;
+
+    return (
+      <div style={styles.header}>
+        <h1 style={styles.h1}> {i18n.feedbackCommentAreaHeader()} </h1>
+        {keepWorkingEnabled && (
+          <TeacherFeedbackKeepWorking
```

Comment:
(minor and for future)  I'm kind of curious to see how we can tweak the names when we move all of these components into their own subfolder.  It might be nice if `TeacherFeedbackKeepWorking` and `FeedbackStudentReviewState` were named in a way that indicated that they were related.

Reply:
Yes! I'd really like to remove the `TeacherFeedback` and `Feedback` prefixes when I move all of the components into a directory for feedback!